README.md

```diff
@@ -17,18 +17,130 @@ The packages will aim to support all current .NET versions. Refer to the current
+        EvaluationContext context = null,
+        FlagEvaluationOptions config = null)
+    {
+        // code to resolve string details
+    }
+
+    public Task<ResolutionDetails<int>> ResolveIntegerValue(string flagKey, int defaultValue,
+        EvaluationContext context = null,
+        FlagEvaluationOptions config = null)
+    {
+        // code to resolve integer details
+    }
+
+    public Task<ResolutionDetails<double>> ResolveDoubleValue(string flagKey, double defaultValue,
+        EvaluationContext context = null,
+        FlagEvaluationOptions config = null)
+    {
+        // code to resolve integer details
+    }
+
+    public Task<ResolutionDetails<T>> ResolveStructureValue<T>(string flagKey, T defaultValue,
+        EvaluationContext context = null,
+        FlagEvaluationOptions config = null)
+    {
+        // code to resolve object details
+    }
+}
+```
+
+### Hook
+
+Hooks are a mechanism that allow for the addition of arbitrary behavior at well-defined points of the flag evaluation life-cycle. Use cases include validation of the resolved flag value, modifying or adding data to the evaluation context, logging, telemetry, and tracking.
+
+Example of adding a hook
+
+```csharp
+// add a hook globally, to run on all evaluations
+openFeature.AddHooks(new ExampleGlobalHook());
+
+// add a hook on this client, to run on all evaluations made by this client
+var client = OpenFeature.Instance.GetClient();
+client.AddHooks(new ExampleClientHook());
+
+// add a hook for this evaluation only
+var value = await client.GetBooleanValue("boolFlag", false, context, new FlagEvaluationOptions(new ExampleInvocationHook()));
+```
+
+Example of implementing a hook
+
+```csharp
+public class MyHook : Hook
+{
+  public Task<EvaluationContext> Before<T>(HookContext<T> context,
+      IReadOnlyDictionary<string, object> hints = null)
+  {
+    // code to run before flag evaluation
+  }
+
+  public virtual Task After<T>(HookContext<T> context, FlagEvaluationDetails<T> details,
+      IReadOnlyDictionary<string, object> hints = null)
+  {
+    // code to run after successful flag evaluation
+  }
+
+  public virtual Task Error<T>(HookContext<T> context, Exception error,
+      IReadOnlyDictionary<string, object> hints = null)
+  {
+    // code to run if there's an error during before hooks or during flag evaluation
+  }
+
+  public virtual Task Finally<T>(HookContext<T> context, IReadOnlyDictionary<string, object> hints = null)
+  {
+    // code to run after all other stages, regardless of success/failure
+  }
+}
+```
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to the OpenFeature project.
+
+Our community meetings are held fortnightly on Thursdays at 4pm. Check the [OpenFeature community calendar](https://calendar.google.com/calendar/u/0?cid=MHVhN2kxaGl2NWRoMThiMjd0b2FoNjM2NDRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for specific dates and for the Zoom meeting links.

```

Comment:
```suggestion
Our community meetings are held regularly and open to everyone. Check the [OpenFeature community calendar](https://calendar.google.com/calendar/u/0?cid=MHVhN2kxaGl2NWRoMThiMjd0b2FoNjM2NDRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for specific dates and for the Zoom meeting links.

```

I would recommend not including specific days and/or times since it may change in the future.